string connection_id
---
bool found
connection_status status
